waku/waku_api/rest/doc/filterapi_v1_subscriptions.yaml

```diff
@@ -0,0 +1,71 @@
+# Legacy support for v1 waku filter
+# /filter/v1/subscriptions:
+post: # post_waku_v2_filter_v1_subscription
```

Comment:
It will be removed yes, probably together with json-rpc counterpart.
I think there can only be users of json-rpc filter v1 and none for rest api filter v1.